client/post-editor/editor-delete-post/index.jsx

```diff
@@ -45,6 +45,7 @@ const EditorDeletePost = React.createClass( {
 
 		const handleTrashingPost = function( error ) {
 			if ( error ) {
+				this.props.onTrashingPost( error );
```

Comment:
I think we ought to just include the `error` argument in the call to `this.props.onTrashingPost` 3 lines below this one and remove the `return`.

Example:

```js
const handleTrashingPost = function( error ) {
	if ( error ) {
		this.setState( { isTrashing: false } );
	}

	this.props.onTrashingPost( error );
}.bind( this );
```